database/migrations/default/1743441451777_noncr3_soft_del_dedupe/up.sql

```diff
@@ -0,0 +1,56 @@
+--
+-- add fk column to atd_apd_blueform.case_id
+--
+alter table ems__incidents add column atd_apd_blueform_case_id integer;
+alter table ems__incidents
+add constraint ems__incidents_atd_apd_blueform_case_id_fkey foreign key (
+    atd_apd_blueform_case_id
+)
+references public.atd_apd_blueform (
+    case_id
+) on update cascade on delete set null;
+
+comment on column ems__incidents.atd_apd_blueform_case_id is 'The non-CR3 case ID matched to this record';
```

Comment:
ah yeah, i left this in here as prep work for https://github.com/cityofaustin/atd-data-tech/issues/22042